django/utils/url_config.py

```diff
@@ -0,0 +1,135 @@
+import inspect
```

Comment:
Would it make more sense to have `url_config` go in `django/conf/`? If you do that, `url_config` might be a confusing name. Maybe something like `services_url_config`? I'm trying to think of something (like "services") that captures both DBs and caches, and the fact that they might be hosted somewhere else.

Reply:
Yeah I think you're right, I think it should be under `conf`.  `service_url` sounds like a good module name I think?